python/pyspark/sql/connect/readwriter.py

```diff
@@ -37,7 +37,7 @@
     from pyspark.sql.connect._typing import ColumnOrName, OptionalPrimitiveType
     from pyspark.sql.connect.session import SparkSession
 
-__all__ = ["DataFrameReader", "DataFrameWriter"]
+__all__ = ["DataFrameReader", "DataFrameWriter", "OptionUtils", "to_str"]
```

Comment:
I don't think we need this change. `OptionUtils` and `to_str` aren't API. They can still be imported without this at `python/pyspark/sql/connect/streaming/readwriter.py`.